examples/carbon-relay-ng.ini

```diff
@@ -58,6 +58,72 @@ init = [
      #'addRoute sendFirstMatch analytics regex=(Err/s|wait_time|logger)  graphite.prod:2003 prefix=prod. spool=true pickle=true  graphite.staging:2003 prefix=staging. spool=true pickle=true'
 ]
 
+# Blacklist
+
+blacklist = [
+  'prefix collectd.localhost',
+  'regex ^foo\..*\.cpu+'
+]
+
+# Aggregations
+
+[[aggregation]]
+type = 'sum'
+regex = '^stats\.timers\.(app|proxy|static)[0-9]+\.requests\.(.*)'
+format = 'stats.timers._sum_$1.requests.$2'
+interval = 10
+wait = 20
+
+[[aggregation]]
+type = 'avg'
+regex = '^stats\.timers\.(app|proxy|static)[0-9]+\.requests\.(.*)'
+format = 'stats.timers._avg_$1.requests.$2'
+interval = 5
+wait = 10
+
+# Rewriters
+
+[[rewriter]]
+old = 'testold'
+new = 'testnew'
+max = -1
+
+# Routes
+
+[[route]]
+key = 'carbon-default'
+type = 'sendAllMatch'
+# prefix = ''
+# substr = ''
+# regex = ''
+destinations = [
+  '127.0.0.1:2003 spool=true pickle=false'
```

Comment:
the way we pass destination options, and blacklist type, as fields in a string is a bit weird. do you plan to also structure these as fields in the future, or do you think we should keep it like this?
ok for me either way I think, this way also keeps the config file more concise.